go.mod

```diff
@@ -7,7 +7,7 @@ replace (
 	github.com/Mirantis/cri-dockerd => github.com/k3s-io/cri-dockerd v0.3.2 // k3s/release-1.27
 	github.com/cloudnativelabs/kube-router/v2 => github.com/k3s-io/kube-router/v2 v2.0.1-0.20230411195838-cced939a8ba1
 	github.com/containerd/cgroups => github.com/containerd/cgroups v1.0.1
-	github.com/containerd/containerd => github.com/k3s-io/containerd v1.5.18-k3s1
+	github.com/containerd/containerd => github.com/k3s-io/containerd v1.6.20-k3s1
```

Comment:
Would be great if this worked; historically this has caused problems so we are still on 1.5 for go.mod and only bump the version in versions.sh. See the background at https://github.com/k3s-io/k3s/blob/master/docs/adrs/standalone-containerd.md

Reply:
If the situation has changed we should update the ADR and maybe try building containerd into the main multicall binary again.

Reply:
Yeah the opentelemetry library got bumped back in 1.26-alpha https://github.com/kubernetes/kubernetes/commit/5be80c05c423daf4bd2a4eb1cabf61fae9a03629, so hopefully this consolidation can also be backported to `release-1.26`

Reply:
The rip-out is at https://github.com/k3s-io/k3s/pull/4757 if you want to try to pull some of that back in.

Reply:
That no longer seems to be a check, all the tests are passing and containerd builds and runs just fine.